packages/wrangler/src/__tests__/type-generation.test.ts

```diff
@@ -603,6 +603,35 @@ describe("generateTypes()", () => {
 	`);
 	});
 
+	it("should respect the loose-vars option", async () => {
```

Comment:
I was considering adding a test for vars generation without loose-vars, but since that's the default behavior (and given the `"vars present in multiple environments"` describe block) that felt a bit overkill here 😕 I'm happy to do it if someone thinks that that would be good

Reply:
actually given the `"vars present in multiple environments"` describe block, this test might be a bit overkill... I'm also happy to remove it entirely 🙂